templating/config.go

```diff
@@ -5,8 +5,8 @@ package templating
 
 // Config represents configuration values.
 type Config struct {
-	RawValue string            `json:"rawValue,omitempty"`
```

Comment:
I assume you never used the json tag and just clean it up here?

Reply:
Correct